src/System.Data.Common/src/Resources/Strings.resx

```diff
@@ -418,9 +418,9 @@
   <data name="ADP_NoQuoteChange" xml:space="preserve"><value>The QuotePrefix and QuoteSuffix properties cannot be changed once an Insert, Update, or Delete command has been generated.</value></data>
   <data name="ADP_MissingSourceCommand" xml:space="preserve"><value>The DataAdapter.SelectCommand property needs to be initialized.</value></data>
   <data name="ADP_MissingSourceCommandConnection" xml:space="preserve"><value>The DataAdapter.SelectCommand.Connection property needs to be initialized;</value></data>
-  <data name="ADP_InvalidMultipartName" xml:space="preserve"><value>{0} "{1}".</value></data>
-  <data name="ADP_InvalidMultipartNameQuoteUsage" xml:space="preserve"><value>{0} "{1}", incorrect usage of quotes.</value></data>
-  <data name="ADP_InvalidMultipartNameToManyParts" xml:space="preserve"><value>{0} "{1}", the current limit of "{2}" is insufficient.</value></data>
+  <data name="ADP_InvalidMultipartName" xml:space="preserve"><value>{0} '{1}'.</value></data>
+  <data name="ADP_InvalidMultipartNameQuoteUsage" xml:space="preserve"><value>{0} '{1}', incorrect usage of quotes.</value></data>
```

Comment:
I wonder whether this and maybe the one above should keep double quotes, since it relates to quoting, and perhaps double quotes were chosen for clarity

Reply:
```
Name: 'ADP_InvalidMultipartName' value: '{0} "{1}".' relative path: 'System.Data.Common\src\Resources\Strings.resx'
Name: 'ADP_InvalidMultipartName' value: '{0} '{1}'.' relative path: 'System.Data.Odbc\src\Resources\Strings.resx'
Name: 'ADP_InvalidMultipartName' value: '{0} '{1}'.' relative path: 'System.Data.SqlClient\src\Resources\Strings.resx'
Name: 'ADP_InvalidMultipartNameQuoteUsage' value: '{0} "{1}", incorrect usage of quotes.' relative path: 'System.Data.Common\src\Resources\Strings.resx'
Name: 'ADP_InvalidMultipartNameQuoteUsage' value: '{0} '{1}', incorrect usage of quotes.' relative path: 'System.Data.Odbc\src\Resources\Strings.resx'
Name: 'ADP_InvalidMultipartNameQuoteUsage' value: '{0} '{1}', incorrect usage of quotes.' relative path: 'System.Data.SqlClient\src\Resources\Strings.resx'
Name: 'ADP_InvalidMultipartNameToManyParts' value: '{0} "{1}", the current limit of '{2}' is insufficient.' relative path: 'System.Data.Common\src\Resources\Strings.resx'
Name: 'ADP_InvalidMultipartNameToManyParts' value: '{0} '{1}', the current limit of '{2}' is insufficient.' relative path: 'System.Data.Odbc\src\Resources\Strings.resx'
Name: 'ADP_InvalidMultipartNameToManyParts' value: '{0} '{1}', the current limit of '{2}' is insufficient.' relative path: 'System.Data.SqlClient\src\Resources\Strings.resx'
```

single on other namespace of same "area".

Reply:
OK